bin/jaxb2ruby

```diff
@@ -81,5 +81,5 @@ begin
     File.open(File.join(outdir, klass.path), "w") { |io| io.puts template.build(klass) }
   end
 rescue => e
-  abort "class generation failed: #{e}"
+  abort "class generation failed: #{e} \n#{'*' * 100}\nbacktrace: \n#{ e.backtrace.join("\n") }"
```

Comment:
It just a fix for debugging. Surely, implementing a debug option would be better :smile: